client/packages/common/src/intl/locales/en/programs.json

```
@@ -10,6 +11,7 @@
   "control.search.no-results-label": "No results",
   "control.search.reset-button": "Unlink matched patient",
   "control.search.searching-label": "Searching...",
+  "error.no-rnr-forms": "No RnR Forms found",
```

```suggestion
  "error.no-rnr-forms": "No R&R Forms found",
```
Not sure if it matters, but thinking we want consistncy?

---

for sure thank you!